src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        return new Ball(ColorSupplier.getRandomColor(), new Random().nextInt(100));

```

Comment:
Don’t use static methods in your solution
Static methods are in general a bad practice. Let’s better create an instance of a class which method you want to call. Think about which variables should be local in the method and which should be class-level.